jobs/payment-jobs/tests/jobs/test_eft_task.py

```diff
@@ -237,6 +266,11 @@ def test_reverse_electronic_funds_transfers(session):
     assert not eft_historical.hidden
     assert not eft_historical.is_processing
 
+    assert refund_requested_invoice.invoice_status_code == InvoiceStatus.REFUNDED.value
```

Comment:
need to check CIL status code `PENDING_REFUND` -> `REFUNDED`
`CANCELLED` shouldn't change